### DISCUSSION TITLE
Responsive Player

### DISCUSSION THREAD

**Discussion by sasicodes** [livepeer/livepeer.js#239] [Ideas]
can we shrink based on size of the player? Its upto discussion 🙇🏻 

<img width="357" alt="image" src="https://user-images.githubusercontent.com/29498872/211843645-72934720-7511-4d19-b0c6-044be62b29fe.png">

**Comment by sasicodes** [comment]
cc @0xcadams

**Reply by 0xcadams** [reply]
What screen size does this UI break at? We've tested down to ~300px which is among the lowest end of mobile screen widths.

I would recommend for this, customizing the `fontSizes.timeFontSizeSm` and `fontSizes.titleFontSizeSm`, and see if that solves this issue.

**Reply by sasicodes** [reply]
https://github.com/lenstube-xyz/lenstube/issues/527

This one is from embedding a video player, and resizing it to a minimum size in notion makes it overflow!

**Reply by sasicodes** [reply]
I will try `timeFontSizeSm` 🙇🏻  do icons can be scaled down?

**Reply by 0xcadams** [reply]
> I will try `timeFontSizeSm` 🙇🏻 do icons can be scaled down?

Currently, no, we can add a field for that in the config!

**Reply by 0xcadams** [reply]
https://github.com/livepeer/livepeer.js/issues/244
